src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -631,25 +742,28 @@ else if ( NoteLine.contains(CuDisbursementVoucherConstants.DV_EXTRACT_NOTE_PREFI
                                         NoteLine = NoteLine.substring(2);
-                                        FirstNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
-                                        
+                                        stripLeadingSpace(NoteLine);
+                                        SecondNoteAfterAddressInfo = stripLeadingSpace(obtainNoteLineSectionExceedingCheckStubLine(NoteLine, FirstNoteAfterAddressInfo));
+                                        FirstNoteAfterAddressInfo = stripTrailingSpace(FirstNoteAfterAddressInfo.concat(obtainLeadingNoteLineSection(NoteLine, FirstNoteAfterAddressInfo)));
                                         // See if we have a second user typed note line.  If so, then get it.
                                         if (ix.hasNext()) {
                                             note = (PaymentNoteText) ix.next();
                                             NoteLine = note.getCustomerNoteText();
                                             if (NoteLine.length() >=2) {
                                                 if (NoteLine.substring(0,2).contains(CuDisbursementVoucherConstants.DV_EXTRACT_TYPED_NOTE_PREFIX_IDENTIFIER)) {
                                                     NoteLine = NoteLine.substring(2);
-                                                    SecondNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
-                                            
+                                                    stripLeadingSpace(NoteLine);
+                                                    ThirdNoteAfterAddressInfo = stripLeadingSpace(obtainNoteLineSectionExceedingCheckStubLine(NoteLine, SecondNoteAfterAddressInfo));
+                                                    SecondNoteAfterAddressInfo = stripTrailingSpace(SecondNoteAfterAddressInfo.concat(obtainLeadingNoteLineSection(NoteLine, SecondNoteAfterAddressInfo)));
                                                     // Try to get the third user typed note line
                                                     if (ix.hasNext()) {
                                                         note = (PaymentNoteText) ix.next();
                                                         NoteLine = note.getCustomerNoteText();
                                                         if (NoteLine.length() >=2) {
                                                             if (NoteLine.substring(0,2).contains(CuDisbursementVoucherConstants.DV_EXTRACT_TYPED_NOTE_PREFIX_IDENTIFIER)) {
                                                                 NoteLine = NoteLine.substring(2);
-                                                                ThirdNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
+                                                                stripLeadingSpace(NoteLine);
```

Comment:
Another no-op stripLeadingSpace() call that may need to be adjusted.

Reply:
See above comment.